### ISSUE TITLE
Unused variable triggered if used *only* in literal array initializer

### ISSUE THREAD

**Issue by danzou56** [Algomorph/CodeGrader#113]
**Describe the bug**
If a variable is used in an array, and only in the array, it is marked unused. E.g.:

```java
int a = 5; // unused
int b = 6; // unused
int[] arr = {a, b}; // but a, b were used here!
```

**Where did the bug occur**
Instructor & Course: Pedram's CMSC131
Project name on Submit Server: Proj 7 Poker
Student directory id: avanbala

**Desktop (please complete the following information):**
 - OS: Windows

**Comment by danzou56** [comment]
Not because it's actually helpful, but because I think it's amusing, here's an image of the problem:

**Comment by Algomorph** [comment]
Thanks for reporting! This is a FP in unuzed code module. Will fix soon!

**Comment by Algomorph** [comment]
Fixed by #118
